.travis.yml

```diff
@@ -101,7 +101,7 @@ script:
   - make -j$(nproc) check
   - cat test-suite.log
   - make -j$(nproc) distcheck
-  - cat config.log $(find -name test-suit.log) || true
+  - cat test-suite.log config.log
```

Comment:
Hmmm... this used to fail with a test-suit.log not found on travis...
Also I wanted to get the test-suit.log from the dist-check directory, not the current directory...

Reply:
Ah, got it - if `make distcheck` is successful, the `test-suite.log` is cleaned by `make distclean`, so there is nothing to show at all (well, the log from `make check` is still there, but we don't need to display it twice). If the distcheck fails and the file is not cleaned up, it is in `tpm2-totp-$PACKAGE_VERSION/_build/sub/`, which is what I `cat` in the updated commit.